README.md

```diff
@@ -36,134 +36,134 @@ pod "JustTweak"
+let configurations: [Configuration] = [userDefaultsConfiguration,
+optimizelyConfiguration,
+firebaseConfiguration,
+localConfiguration]
+tweakManager = TweakManager(configurations: configurations)
 }
 ```
 
-The order of the objects in the `configurations` array defines the priority of the configurations. The `MutableTweaksConfiguration` with the highest priority, such as `UserDefaultsTweaksConfiguration` in the example above, will be used to load the `TweaksConfigurationViewController` UI. The `JSONTweaksConfiguration` should have the lowest priority as it provides the default values from a local configuration.
+The order of the objects in the `configurations` array defines the priority of the configurations. The `MutableConfiguration` with the highest priority, such as `UserDefaultsConfiguration` in the example above, will be used to load the `TweakViewController` UI. The `LocalConfiguration` should have the lowest priority as it provides the default values from a local configuration.
 
 
 ### Usage
 
-The three main features of JustTweak can be accessed from the `TweaksConfigurationsCoordinator` instance to drive code path decisions.
+The three main features of JustTweak can be accessed from the `JustTweak` instance to drive code path decisions.
 
 1. Checking if a feature is enabled
 
 ```swift
 // check for a feature to be enabled
-let enabled = configurationsCoordinator.isFeatureEnabled("some_feature")
+let enabled = tweakManager.isFeatureEnabled("some_feature")
 if enabled {
-    // enable the feature
+// enable the feature
 } else {
-    // default behaviour
+// default behaviour
 }
 ```
 
 2. Get the value of a flag for a given feature. JustTweak will return the value from the configuration with the highest priority and automatically fallback to the others if no set value is found.
 
-Use either `tweakWith(feature:variable:)` or the provided property wrappers: `@FeatureFlag` and `@FeatureFlagWrappingOptional`.
+Use either `tweakWith(feature:variable:)` or the provided property wrappers: `@FeatureFlag` and `@OptionalTweak`.
 
 ```swift
 // check for a tweak value
-let tweak = configurationsCoordinator.tweakWith(feature: "some_feature", variable: "some_flag")
+let tweak = tweakManager.tweakWith(feature: "some_feature", variable: "some_flag")
 if let tweak = tweak {
-    // tweak was found in some configuration, use tweak.value
+// tweak was found in some configuration, use tweak.value
 } else {
-    // tweak was not found in any configuration
+// tweak was not found in any configuration
 }
 ```
 
-`@FeatureFlag` and `@FeatureFlagWrappingOptional` are available to mark properties representing feature flags. Mind that by using these property wrappers, a static instance of `TweaksConfigurationsCoordinator` must be available.
+`@FeatureFlag` and `@OptionalTweak` are available to mark properties representing feature flags. Mind that by using these property wrappers, a static instance of `JustTweak` must be available.
```

Comment:
I think you mean `@TweakProperty`?

Reply:
I was just updating the README when you posted the comment :)